validation.go

```diff
@@ -0,0 +1,69 @@
+package epp
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/lestrrat-go/libxml2"
+	xsd "github.com/lestrrat-go/libxml2/xsd"
+)
+
+// Validator represents a validator holding the XSD schema to calidate against.
+type Validator struct {
+	Schema *xsd.Schema
+}
+
+// NewValidator creates a new validator.
+func NewValidator(rootXSD string) (*Validator, error) {
```

Comment:
Perhaps we can provide the data directly instead? That way the xsd can theoretically be included in the binary instead of loaded from a file.

Reply:
Sounds like a good idea but since the XML parser does something with the files I'm not sure about we need to investigate! I tried to construct a single XSD file (which could be embedded in the binary) but that didn't work out of the box. Let's add an issue for that matter!